test/Riok.Mapperly.Tests/Mapping/ExtensionMethodTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Riok.Mapperly.Tests.Mapping;
+
+[UsesVerify]
+public class ExtensionMethodTest
+{
+    [Fact]
+    public Task UpdateMethod()
```

Comment:
`UpdateMethod` is not really a descriptive name for a test...
Probably `ExtensionMethodShouldWork` or similar should fit better.